### DISCUSSION TITLE
A way to control Apple Cinema Display 20" model A1038 brightness

### DISCUSSION THREAD

**Discussion by mbirafon** [MonitorControl/MonitorControl#1080] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Read through [the Monitor-Troubleshooting Wiki](https://github.com/MonitorControl/MonitorControl/wiki/Monitor-Troubleshooting)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the issue

Hello, 
is there a way to control Apple Cinema Display's brightness from Mac OS High Sierra running on a Mac mini 2012?
The display is an old **Apple Cinema Display 20" model A1038 Clear Enclosure**. 
I tried to install Snow Leopard Server on Parallel Desktop 13, but the brightness cursor in Monitor preferences doesn't show up. 
Furthermore, Monitor Control 2.1 utility doesn't work with the Apple Cinema Display 20" model A1038. 
Any suggestions? 
Thanks.

### Expected behavior

1) Monitor Control 2.1 utility doesn't work with the Apple Cinema Display 20" model A1038.
2) In the virtualized Mac OSX Snow Leopard, the brightness cursor in Monitor preferences doesn't show up.

### Anything else?

![missing brighness cursor](https://user-images.githubusercontent.com/106884984/172966522-2fef30f7-0b6f-4ab8-b590-c8f30c455e66.png)

### Environment Information (please complete the following information)

```markdown
- macOS version: 10.13 High Sierra
- Mac model: Mac mini 2012
- MonitorControl version: 2.1
- Monitor(s): Apple Cinema Display 20" model A1038 Clear Enclosure.
- Monitor Cable(s)/Connection(s): ADC-DVI-Thunderbolt
- Apple Silicon/M1 (yes or no): No
```

**Comment by waydabber** [comment]
I am afraid this won't be possible. As far as I know the display requires a combination of USB connection and native display detection. These are certainly obfuscated by the virtual machine environment (even if you could route the direct USB connection, the direct DVI connection won't be there, maybe if you could somehow emulate an EDID towards the virtual machine)?

I'll convert this into a discussion, maybe more clever people can say something about it. :)
